mne_bids_pipeline/tests/configs/config_ds003392.py

```diff
@@ -22,7 +22,6 @@
 ica_algorithm = "picard-extended_infomax"
 ica_max_iterations = 1000
 ica_l_freq = 1.0
-ica_n_components = 0.99
```

Comment:
maybe we should keep one example where we pass a float and one where we pass an int?

Reply:
👍 good idea. I'll revert or put some in... will let it finish running with always using the new default to make sure it doesn't break anything first though.

Reply:
thanks!